src/Foliage.rb

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env ruby
+
+class Foliage
```

Comment:
Need documentation for class

Reply:
https://github.com/Wall-Dough/pointless/pull/5/commits/0f120a68c273ea0a3ccb3a6602fff227e31995d5